image/exposure_histogram/histogram.py

```diff
@@ -0,0 +1,216 @@
+
+from warnings import warn
+import tensorflow as tf
+
+
+"""

```

Comment:
Good block, usually best at very start of file unless it is doc for a class etc.